packages/node_modules/@node-red/editor-client/src/js/ui/view-tools.js

```diff
@@ -105,6 +105,8 @@ RED.view.tools = (function() {
                 $(document).one('keyup',endKeyboardMove);
                 endMoveSet = true;
             }
+            var space_width = 5000;
+            var space_height = 5000;
```

Comment:
Would it be better for `RED.view` to expose these values in a function, such as `RED.view.dimensions()` which returns `{width: 5000, height: 5000}` - that way these values are defined in one place.

Reply:
Thank you for your suggestion.
Changed the code to use RED.view.dimensions.